themes/custom/az_barrio/az_barrio.libraries.yml

```diff
@@ -9,6 +9,10 @@ arizona-bootstrap:
       /libraries/arizona-bootstrap/css/arizona-bootstrap.min.css: {}
   js:
     /libraries/arizona-bootstrap/css/arizona-bootstrap.bundle.min.js: {}
+  dependencies:
```

Comment:
We might want to add these dependencies via hook_library_info_alter based on whether the font and icons are enabled.

Reply:
I think that's a good idea.

Reply:
OK I've updated the PR

Reply:
See https://github.com/az-digital/az_quickstart/pull/335/files#diff-31e694b1bcef5ef225b34a8cbb592743b8511bf563f242544cdaa4db9193fb27R317-R323

Reply:
And see https://github.com/az-digital/az_quickstart/pull/335/files#diff-31e694b1bcef5ef225b34a8cbb592743b8511bf563f242544cdaa4db9193fb27R296 for brand icons